src/encoded/tests/features/test_audit.py

```diff
@@ -8,6 +8,13 @@
     pytest.mark.usefixtures('workbook'),
 ]
 
+
+def test_auditview(testapp, workbook):
```

Comment:
confused by this as well

Reply:
A space?  You should look at the commit history.